WhyNotWin11.au3

```diff
@@ -533,13 +534,22 @@ Func Main()
 			GUICtrlSetData($hCheck[8][2], _Translate($iMUI, "Disabled / Not Detected"))
 	EndSwitch
 
-	_SpaceCheck()
-	GUICtrlSetData($hCheck[9][2], @error & " GB " & $WINDOWS_DRIVE & @CRLF & @extended & " " & _Translate($iMUI, "Drive(s) Meet Requirements"))
-	If _SpaceCheck() Then
-		_GUICtrlSetPass($hCheck[9][0])
-	Else
-		_GUICtrlSetFail($hCheck[9][0])
+	#Region - SpaceCheck
+	$sSpaceResultFirstLine = _SpaceCheck(3) & " " & _SpaceCheck(4) & " GB, " & _Translate($iMUI, "Disk") &  " " & _SpaceCheck(1) & ": " & _SpaceCheck(2) & " GB"
+	If _SpaceCheck(4) >=  64 And _SpaceCheck(2) >= 64 Then
+		; Partition and Disk >= 64 GB
+		_GUICtrlSetState($hCheck[9][0], $iPass)
+		GUICtrlSetData($hCheck[9][2], $sSpaceResultFirstLine & @CRLF & _Translate($iMUI, "Both passed."))
```

Comment:
Partition C: (xyz GB): passed
Disk 0 (xyz GB): not passed

or

Disk: passed, Partition passed